src/routes/safe/components/Apps/hooks/useAppList.ts

```diff
@@ -40,12 +42,13 @@ const useAppList = (): UseAppListReturnType => {
         try {
           const currentApp = list[index]
 
-          const appInfo: any = await getAppInfoFromUrl(currentApp.url)
+          const appInfo: SafeApp = await getAppInfoFromUrl(currentApp.url)
           if (appInfo.error) {
             throw Error(`There was a problem trying to load app ${currentApp.url}`)
           }
 
-          appInfo.disabled = currentApp.disabled === undefined ? false : currentApp.disabled
+          appInfo.disabled = Boolean(currentApp.disabled)
+          appInfo.isDeletable = !staticAppsList.some((staticApp) => staticApp.url === currentApp.url)
```

Comment:
Can't we just set isDeletable: false for apps in staticAppsList?

Reply:
@mikheevm  Do you mean in utils.ts? where we set this: 
```
  { url: `${process.env.REACT_APP_IPFS_GATEWAY}/QmUDTSghr154kCCGguyA3cbG5HRVd2tQgNR7yD69bcsjm5`, disabled: false, isDeletable: false },
```

Reply:
Yep

Reply:
I reimplemented it in another way